### ISSUE TITLE
test user not functional

### ISSUE THREAD

**Issue by garloff** [osism/cloud-in-a-box#17]
The Cloud-in-a-Box install has a test user in `/opt/configuration/environments/openstack/clouds.yml`
(which is used by the openstackclient container).

However, that user does not seem to be configured in the deployed openstack keystone ...
So we should probably create a test project and assign the test user as tenant user, no?

**Comment by garloff** [comment]
I'm struggling with setting up the external/public network, so I can get access to the internet from the VMs (via SNAT or FIPs). Trying to use physnet1 as flat provider network -- but it's unclear to me how physnet1 relates to the physical NICs / VLANs. I guess I need to better understand the ml2 config? ovn?

**Comment by berendt** [comment]
```vlan100``` is the internal interface. ```vlan101``` is the external interface. ```vlan101``` is mapped to ```physnet1```.

Try if the approach we are using in the testbed is working in the cloud-in-a-box as well. There we have a vxlan instead of a vlan.

```
sysctl -w net.ipv4.ip_forward=1
iptables -A FORWARD -i eno1 -j ACCEPT
iptables -A FORWARD -o eno1 -j ACCEPT
iptables -t nat -A POSTROUTING -o eno1 -j MASQUERADE
```

**Comment by tibeer** [comment]
The openstack client works if you export the OS_CLOUD environment variable on the host. This get's passed automatically into the container. Will add this to the documentation.

**Comment by berendt** [comment]
* test user/project fixed with 80663abd6d3857e93511579765cded56bd80fcde
